src/SearchResults.js

```diff
@@ -1,11 +1,22 @@
-import React from "react";
+import React, { useState } from "react";
 import moment from "moment";
 
 function SearchResults(props) {
+  // const [clickRow, setClickRow ] = useState(true) - l16 try?
+
+  // function RowClick(){
+  //   if (clickRow===true){
+  //   {'tr {background-color: red; }'}
+  //     setClickRow(false)
+  //   } else {
+  //     setClickRow(true)
+  //   }
+  // }
```

Comment:
This is a good attempt, but it looks like you realised that if you click on any row, then all of them become highlighted.

We'll look at a simple solution on Sunday, but the idea is to think carefully about how we could split up our components so that they can have their own state. Can you figure out how to fix it?